sktime/param_est/seasonality.py

```diff
@@ -141,7 +141,7 @@ def _fit(self, X):
 
         candidate_sp = self.candidate_sp
         if candidate_sp is None:
-            candidate_sp = range(2, nlags + 1)
+            candidate_sp = range(2, nlags)
```

Comment:
Why is this being reverted back to nlags? nlags + 1 was added for a specific reason?

@fkiraly FYI...

Reply:
uh-oh, I think this is just an accident, probably from file copy-paste.
Should be reverted.